src/gql/queries/get-patch-tasks.graphql

```diff
@@ -42,6 +43,7 @@ query PatchTasks(
         baseStatus
         buildVariantDisplayName
         projectIdentifier
+        taskDuration
```

Comment:
I do show them in the table! (They are an expandable row, one is actually viewable on the big patch on `dev`)

I thought it might be helpful for a user to see which one of the execution tasks is taking really long, if the display task is taking very long. I can remove it though 😲